app/src/main/java/tech/zafrani/pubgapp/adapters/viewholders/AttachmentsViewHolder.java

```diff
@@ -0,0 +1,141 @@
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.HashMap;
+
+import tech.zafrani.pubgapp.R;
+import tech.zafrani.pubgapp.models.Item;
+
+public class AttachmentsViewHolder extends ItemViewHolder {
+    private final TextView nameTextView;
+    private final TextView typeTextView;
+    private final TextView categoryTextView;
+    private final TextView magazineTextView;
+    private final TextView capacityTextView;
+    private final TextView damageTextView;
+    private final TextView rangeTextView;
+    private final TextView stabilityTextView;
+    private final TextView rateTextView;
+    private final TextView reloadTextView;
+    private final ImageView imageImageView;
+    private final Context context;
+
+    public AttachmentsViewHolder(@NonNull final View itemView, @NonNull final Context context) {
+        super(itemView);
+        this.context = context;
+        this.nameTextView = (TextView) itemView.findViewById(R.id.row_item_name);
+        this.typeTextView= (TextView) itemView.findViewById(R.id.row_item_type);
+        this.categoryTextView = (TextView) itemView.findViewById(R.id.row_item_category);
+        this.magazineTextView= (TextView) itemView.findViewById(R.id.row_item_magazine);
+        this.capacityTextView= (TextView) itemView.findViewById(R.id.row_item_capacity);
+        this.damageTextView= (TextView) itemView.findViewById(R.id.row_item_damage);
+        this.rangeTextView= (TextView) itemView.findViewById(R.id.row_item_range);
+        this.stabilityTextView = (TextView) itemView.findViewById(R.id.row_item_stability);
+        this.rateTextView= (TextView) itemView.findViewById(R.id.row_item_rate);
+        this.reloadTextView= (TextView) itemView.findViewById(R.id.row_item_reload);
+        this.imageImageView= (ImageView) itemView.findViewById(R.id.row_item_icon);
+
+
+    }
+
+    private void setItemName(@NonNull final String itemName) {
+        this.nameTextView.setText(itemName);
+    }
+
+    private void setItemType(@NonNull final String itemType) {
+        this.typeTextView.setText(itemType);
+    }
+
+    private void setItemCategory(@NonNull final String itemCategory) {
+        this.categoryTextView.setText(itemCategory);
+    }
+
+
+    private void setItemMagazine(@Nullable final Float itemMagazine) {
+        if (itemMagazine != null) {
+            this.magazineTextView.setText(context.getString(R.string.row_item_magazine, itemMagazine));
+        }else {
+            this.magazineTextView.setText(EMPTY_FIELD);
+        }
+    }
+    private void setItemCapacity(@Nullable final Float itemCapacity) {
+        if (itemCapacity != null) {
+            this.capacityTextView.setText(context.getString(R.string.row_item_capacity, itemCapacity));
+        }else {
+            this.capacityTextView.setText(EMPTY_FIELD);
+        }
+    }
+    private void setItemDamage(@Nullable final HashMap<String,String> itemDamage) {
```

Comment:
This hashmap should be a class that extends HashMap<String,String> and provides a function `getChest0()`